packages/graphql/src/translate/utils/get-mutation-field-statements.ts

```diff
@@ -100,3 +114,66 @@ export function getMutationFieldStatements({
     }
     return strs.join("\n");
 }
+
+// Converts generic operator into cypher statements using the deprecated syntax as intermediate step
+function getMutationFieldStatementsForGenericOperator({
+    nodeOrRel,
+    param,
+    key,
+    varName,
+    operations,
+    withVars,
+}: {
+    nodeOrRel: Node | Relationship;
+    param: string;
+    key: string;
+    varName: string;
+    operations: any;
+    withVars: string[];
+}): string {
+    if (Object.entries(operations).length > 1) {
```

Comment:
I would use `assertNonAmbiguousUpdate` that took care of `@alias` fields as well

Reply:
it is worth adding a test, but I think it will still work, this is an extra level check that 2 fields cannot be inside `title`, but assertNonAmbiguous is still being called to ensure that title + aliases are not being called twice